pvlib/financial.py

```diff
@@ -0,0 +1,116 @@
+import numpy as np
+
+def lcoe(production=None, cap_cost=None, fixed_om=None):
+        """
+        Levelized cost of electricity as described on pp.43 and 47-48 by [1]   
+        Parameters
+        ----------
+        production : np.array, or pd.Series, default None
+            Annual production [kWh/kW installed]
+        cap_cost : np.array, or pd.Series, default None
+            Initial and annual payments on capital costs [$/kW installed]
+        fixed_om : np.array, or pd.Series, default None
+            Annual payments on operations and maintenance costs [$/kW installed]
+            
+        Returns
+        ----------
+        LCOE [cents/kWh]
+
+        References
+        ----------
+        .. [1] W. Short, D. J. Packey, and T. Holt, "A Manual for the Economic Evaluation of Energy Efficiency and Renewable Energy Technologies", NREL/TP-462-5173, 1995.
+        """        
+        if len(production)!=len(cap_cost) or len(cap_cost)!=len(fixed_om):
```

Comment:
I think you should remove these lines. IMO they are unnecessary, and I think they reduce readability. I believe the pythonic way is EAFP instead of LBYL. If capital costs & fixed O&M cannot be dined then bumpy will raise an exception which will yield the cause of the error more effectively than this value error. Ditto for the quotient of cost & production.